hudi-common/src/main/java/org/apache/hudi/common/util/FSUtils.java

```diff
@@ -485,6 +485,14 @@ public static void deleteOlderRollbackMetaFiles(FileSystem fs, String metaPath,
     });
   }
 
+  public static void deleteInstantFile(FileSystem fs, String metaPath, HoodieInstant instant) {
+    try {
+      fs.delete(new Path(metaPath, instant.getFileName()), false);
```

Comment:
Add a Log message to not what is getting deleted ?

Reply:
reasonable